docs/troubleshooting.md

```diff
@@ -0,0 +1,127 @@
+# Troubleshooting Marketplace Operator
+
+This document lists out some common failures related to the Marketplace operator a user might encounter, along with ways to troubleshoot the failures. If you encouter a failure related to the Marketplace operator that is not listed in this document, but should be, please open an issue or a PR to have the failure appended to this document. 
+
+Note that this document lists out troubleshooting steps for Marketplace Operator defined resources like OperatorSource and CatalogSourceConfig. To troubleshoot [Operator-lifecycle-manager(OLM)](https://github.com/operator-framework/operator-lifecycle-manager) defined resources like ClusterServiceVersion, InstallPlan, Subscription etc, refer to the [OLM troubleshooting guide](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Documentation/design/debugging.md).
```

Comment:
I suggest dropping `Note that`. It seems redundant. `The troubleshooting steps listed here are for Marketplace resources like...`